src/services/drive/folder/driveFolder.service.ts

```diff
@@ -68,7 +68,7 @@ class DriveFolderService {
         const fileFullName = `${file.name}${file.type ? '.' + file.type : ''}`;
         const relativePath = `${currentFolder.relativePath}/${fileFullName}`;
 
-        fileService.renameFileInNetwork(file.fileId, bucketId, relativePath);
+        driveFileService.renameFileInNetwork(file.fileId, bucketId, relativePath);
```

Comment:
shouldn't this be - according to what has been done here - ? `drive.file.rename...`